core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5434,6 +5434,14 @@ public String toString() {
           .setDescription("The port Alluxio FUSE web UI runs on.")
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey FUSE_JNIFUSE_VERSION =
+      new Builder(Name.FUSE_JNIFUSE_VERSION)
+          .setDefaultValue(0)
```

Comment:
use enum?

Reply:
Enum's name cannot be number. If it has to be an enum, the conf value has been more complicated (like `fuse2` instead of `2`), which is unnecessary.